### ISSUE TITLE
Get the experimental branch worked into the git repository

### ISSUE THREAD

**Issue by lsolesen** [ole1986/pdf-php#1]
I am working on getting R&OS PDF class to github. Progress can be seen here: https://github.com/lsolesen/pdf-php-svn --> Right now that is the old svn repository with tags and branches. 

Problem right now, is that there is no common commits. Would you do the honors of branching and adding the experimental on top of the current master of the above github-repository? Then the development could be pretty much streamlined. Then you will get commit credit for the changes?

I can also try to merge it in, but then it will be as one single commit on the branch, so there will be no history on the experimental branch. @ole1986 What is your preference?

When we are satisfied with the progress in the above repository, I will move everything to this repository.

**Comment by lsolesen** [comment]
If you want me to do it, I need your git name and git email you use for github, so I can give you commit credit for the commits.

**Comment by ole1986** [comment]
I, just give me 1-2 hour(s)... To much things at the same time going on

**Comment by ole1986** [comment]
@lsolesen all in i guess.. what do you think should we de with sourceforge repo?

**Comment by lsolesen** [comment]
@ole1986 The 0.13-exp branch does not seem to be based on master - it is behind with 97 commits. That means we will not be able to merge them, will we? Because I think 0.13-exp should be merged in as soon as the code is working, so there is only one master branch? There is no need having people supplying pull requests against master, when you think 0.13 is the new main branch? People can always access the old versions and if they think the old stuff is better, they can just fork the repos and work on previous versions.

As to the sourceforge repo, I think we should move over the issues, if you think they are still valid. Then update the website to redirect directly to the github-repo - and remove all code. If there is a mailing list, tell people the repo has moved - and then close it down. After maybe three months remove it entirely from sf? How does that sound?

**Comment by lsolesen** [comment]
@ole1986 Also it seems you took over all commits when you put it in your repos. E.g. I've added the composer.json, but you are now the committer?

**Comment by lsolesen** [comment]
@ole1986 If you see the commits in https://github.com/lsolesen/pdf-php-svn/commits/master I've retained committers, which I think is important for open source projects.

**Comment by lsolesen** [comment]
@ole1986 I can see in the commit log what your author information for git are. Should I do it?

**Comment by ole1986** [comment]
I have now clue what you are talking about :-) but feel free to do what you want to ;-)

**Comment by ole1986** [comment]
but in future when you change files you will be shown as committer I guess

**Comment by ole1986** [comment]
btw what does the composer.json do?

**Comment by ole1986** [comment]
If you have added the composer.json than also describe it in the README.md on how to use it!
regarding the committers I am pretty sure we can keep it know as it is.. and in future others will see who has committed the lines...

**Comment by lsolesen** [comment]
@ole1986 I've fixed it all up now. I've added `experimental` branch instead of your 0.13-exp-branch. Now all commits are correct and you got credit for your commits.

`composer.json` is information for packagist.org which basically can be used as a package manager in PHP. It is in all modern PHP repositories right now. We should add a section in the README.md on how to install the package, and there we could add information about `composer`. Opened an issue here: https://github.com/ole1986/pdf-php/issues/3
